data/json/items/gun/308.json

```diff
@@ -4,8 +4,17 @@
     "copy-from": "rifle_auto",
     "looks_like": "ar15",
     "type": "GUN",
-    "name": { "str_sp": "FN FAL" },
-    "description": "Originally designed during the Cold War, the FN FAL is probably the most successful battle rifle ever designed.  Even though often labeled as obsolete, its high rate of fire and powerful ammunition make it perfectly capable of holding its ground against modern competitors.",
+    "name": { "str_sp": "Cold War battle rifle" },
```

Comment:
When I was adding gun variants like this one, I was always advised to make the names as useful as possible and to pack in enough information that a non-gun-familiar player could know what the weapon was by glancing at it. "Cold War" doesn’t give any useful information, and it can be translated in the description. I’d consider changing the name to something more useful with more utility. You’d also want to add a variant for the FN FAL’s magazines to give them a non-branded description that matches the title of the gun.

Reply:
7.62x51 battle rifle (somewhat vague, i could call G3 or M 14 EBR with this name also)
7.62x51 long-barreled battle rifle (i dont like it, too long, but it most closely reflect what it is. M14 has longer barrel, but i see FN FAL more on "assault" side of battle rifles, and M14 on "high precision" side. )
Belgian battle rifle (also don't like it. This says too little)

Your opinion?

Reply:
> 7.62x51 battle rifle (somewhat vague, i could call G3 or M 14 EBR with this name also) 7.62x51 long-barreled battle rifle (i dont like it, too long, but it most closely reflect what it is. M14 has longer barrel, but i see FN FAL more on "assault" side of battle rifles, and M14 on "high precision" side. ) Belgian battle rifle (also don't like it. This says too little)
> 
> Your opinion?

If you wanted to be a little cheaty, you could always just do a literal translation of "FAL" to "light automatic rifle" and have that be the non-branded name. I wouldn’t recommend it, however, as I would suppose that non-gun people would see the term "light" and assume it to be a weapon chambered in something comparatively small like 5.56 and not 7.62. As such, I would go for "heavy automatic rifle," which leaves room for the term "battle rifle" to be used with other weapons
.

Reply:
> I wouldn’t recommend it, however, as I would suppose that non-gun people would see the term "light" and assume it to be a weapon chambered in something comparatively small like 5.56 and not 7.62

Yes, that the point i dont mentioned it here. 

heavy automatic rifle it is (but G3 weight more... well, it's for those poor soul, who would thinking about G3 brand-less name, i suppose)